exporter/elasticsearchexporter/README.md

```diff
@@ -25,10 +25,18 @@ This exporter supports sending OpenTelemetry logs to [Elasticsearch](https://www
   [index](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices.html)
   or [datastream](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html)
   name to publish events to. The default value is `logs-generic-default`
+- `logs_dynamic_index` (optional): 
+  takes resource or log record attribute named `logs_index_prefix` and `logs_index_suffix`
```

Comment:
> however having a static variable being concatenated is good way to have a fallback in case that defined resource or attribute is not available.

Not sure I understand this concern. Wouldn't the fallback be the same value defined in `logs_index` in both cases? I'm just trying to see if there are strong reasons to provide this rather more complicated interface to users

Reply:
@dmitryax 
we could use instead certain resources/attributes itself as index without prefixing or suffixing `logs_index` and `traces_index` leaving `logs_index` and `traces_index` as defaults, if there are no attributes found.

however this way, there is one downside. it is harder to specify **index pattern** in elasticsearch, as there is no guarantee of having similarity in the resulting index names.